crates/runtime/src/accelerated_table.rs

```diff
@@ -73,22 +110,69 @@ impl AcceleratedTable {
             accelerator,
             federated,
             refresh_handle,
+            refresh_mode,
+            refresh_trigger,
+            _scheduled_refreshes_handle: scheduled_refreshes_handle,
+        }
+    }
+
+    #[allow(dead_code)]
+    pub async fn trigger_refresh(&self) -> Result<()> {
+        if self.refresh_mode == RefreshMode::Append {
```

Comment:
It seems a bit redundant to save the `refresh_mode` to check it here - couldn't we just move this error into the `None` arm below and not need to store `refresh_mode` on the table?

Reply:
That would mean that not having a `refresh_trigger` implies that it is an Append refresh, which I think makes sense here.

Reply:
Updated, thank you (I thought about proper support when there will be more modes, updated as unnecessary at this point)